ansible/library/announce_routes.py

```diff
@@ -930,10 +930,12 @@ def fib_t2_lag(topo, ptf_ip, action="announce"):
     t1_vms = {}
     # T3 VMs per linecard(asic) - key is the dut index, and value is a list of T3 VMs
     t3_vms = {}
-
     for key, value in vms.items():
-        m = re.match(r"(\d+)\.(\d+)@(\d+)", value['vlans'][0])
-        dut_index = int(m.group(1))
```

Comment:
iso of changing the common code, can you define the topo similar to topo_t2 where the vmoffset is of the format `x.y@n`

Reply:
discussed offline. Making this change will complicate the test for disagg chassis